modules/drivers/canbus/sensor_canbus.h

```diff
@@ -200,11 +214,34 @@ void SensorCanbus<SensorType>::OnTimer(const ros::TimerEvent &) {
 }
 
 template <typename SensorType>
+void SensorCanbus<SensorType>::DataTrigger() {
+  std::condition_variable* cvar = sensor_message_manager_->GetMutableCVar();
+  while (data_trigger_running_) {
+    std::unique_lock<std::mutex> lock(mutex_);
+    cvar->wait(lock);
+    // TODO(lizh): this log is for test. Please remove it after onboard test.
+    AINFO << "===== Pulibsh Sensor Data =====";
```

Comment:
Pulibsh -> Publish

Reply:
Done.

Reply:
Btw, this is a log for test only.  Will be removed after onboard test.